src/kron.jl

```diff
@@ -0,0 +1,66 @@
+export opKron
+
+type opKron{T} <: AbstractLinearOperator{T}
```

Comment:
Isn't this the same as `LinearOperator` (https://github.com/JuliaSmoothOptimizers/LinearOperators.jl/blob/master/src/LinearOperators.jl#L50)? Maybe there's no need for an new type here.

Reply:
That's true, hadn't realized that.